android/src/main/java/com/getcapacitor/plugin/http/Http.java

```diff
@@ -391,8 +390,12 @@ private void buildResponse(PluginCall call, HttpURLConnection conn) throws Excep
           JSObject jsonValue = new JSObject(builder.toString());
           ret.put("data", jsonValue);
         } catch (JSONException e) {
-          JSArray jsonValue = new JSArray(builder.toString());
-          ret.put("data", jsonValue);
+            try {
+                JSArray jsonValue = new JSArray(builder.toString());
```

Comment:
Sorry, didn't have the prettier config set up, but can you change anywhere you're using four spaces to use two spaces (no tabs). This will be automated in the future, thanks.